src/App.js

```diff
@@ -10,20 +11,17 @@ import Navigation from "./components/Navigation";
 import Div100vh from "react-div-100vh";
 
 function App() {
-  const [menuItems, setMenuItems] = useState([]);
+  const [menu, setMenu] = useState([]);
+  const [order, setOrder] = usePersistedState("order", []);
 
-  const history = useHistory();
+  let history = useHistory();
```

Comment:
why changing it to let?

Reply:
Actually I was confused... I read that it was let instead of const. Then I changed it to const because I thought that must be true. But then I changed it back to let, because that's what I read. 😅